lib/node_modules/@stdlib/stats/base/dists/uniform/cdf/benchmark/benchmark.js

```diff
@@ -32,16 +33,24 @@ var cdf = require( './../lib' );
 bench( pkg, function benchmark( b ) {
 	var min;
 	var max;
+	var len;
 	var x;
 	var y;
 	var i;
 
+	len = 100;
+	x = new Float64Array(len);
+	min = new Float64Array(len);
+	max = new Float64Array(len);
+	for(i = 0; i < len; i++){
+		x[ i ] = uniform(-10.0, 20.0);
+		min[ i ] = uniform(-20.0, 20.0);
```

Comment:
```suggestion
		x[ i ] = uniform( -10.0, 10.0 );
		min[ i ] = uniform( -20.0, 0.0 );
```

The range remains the same as in the previous code. Also, brackets should have spaces at the beginning and end.